package.json

```diff
@@ -50,12 +50,14 @@
     "react": "^16.13.1",
     "react-app-polyfill": "^0.2.2",
     "react-beautiful-dnd": "^13.0.0",
+    "react-bootstrap": "^1.4.3",
```

Comment:
do you need it?

Reply:
did you run `npm uninstall react-bootstrap` or just removed it yourself?

Reply:
just removed it